operator/cmd/cilium_node.go

```diff
@@ -48,8 +48,11 @@ var (
 	ciliumNodeStore cache.Store
 
 	k8sCiliumNodesCacheSynced = make(chan struct{})
+	ciliumNodeMangerSynced    = make(chan struct{})
 )
 
+type CacheInSync struct{}
```

Comment:
Does this need to be exported?

Also, we could probably do with a more descriptive name? How about
```suggestion
type ciliumNodeManagerSyncedKey struct{}
```